test/common/tests.go

```diff
@@ -84,6 +84,7 @@ var (
 		{
 			[]TestCase{
 				{"M02B - Verify RHOAM version metric is exposed in Prometheus", TestRhoamVersionMetricExposed},
+				{"Validate resource requirements are set", ValidateResourceRequirements},
```

Comment:
If I'm not mistaken, codes (e.g. M03) are assigned based on the existence of a manual test. In this case this would be a brand new test so not sure that this is applicable :thinking: